core/src/test/java/kafka/test/junit/ZkClusterInvocationContext.java

```diff
@@ -248,5 +261,17 @@ public void stop() {
                 clusterReference.get().tearDown();
             }
         }
+
+        @Override
+        public void rollingBrokerRestart() {
+            if (started.get()) {
+                for (int i = 0; i < clusterReference.get().brokerCount(); i++) {
```

Comment:
It seems like there should be a log message here before we kill the broker, to make it easier to see what is going on?